<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::focusingwindow</class>
 <widget class="QWidget" name="snowgui::focusingwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="focusingWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="snowgui::focusingcontrollerwidget" name="focusingcontrollerWidget" native="true"/>
         </item>
         <item>
          <widget class="snowgui::FocusingProgressWidget" name="focusingHistory" native="true"/>
         </item>
         <item>
          <widget class="snowgui::FocusElementStack" name="focusingMonitor">
           <widget class="QWidget" name="focusingMonitorPage1"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="snowgui::imagedisplaywidget" name="imageWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="deviceWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="snowgui::ccdcontrollerwidget" name="ccdcontrollerWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>325</width>
          <height>261</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="separatorLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="snowgui::focusercontrollerwidget" name="focusercontrollerWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>325</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="snowgui::coolercontrollerwidget" name="coolercontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>325</width>
           <height>563</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Cooler</string>
         </attribute>
        </widget>
        <widget class="snowgui::filterwheelcontrollerwidget" name="filterwheelcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Filterwheel</string>
         </attribute>
        </widget>
        <widget class="snowgui::guideportcontrollerwidget" name="guideportcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Guide Port</string>
         </attribute>
        </widget>
        <widget class="snowgui::adaptiveopticscontrollerwidget" name="adaptiveopticscontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Adaptive Optics</string>
         </attribute>
        </widget>
        <widget class="snowgui::mountcontrollerwidget" name="mountcontrollerWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Mount</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::imagedisplaywidget</class>
   <extends>QWidget</extends>
   <header location="global">imagedisplaywidget.h</header>
   <container>1</container>
   <slots>
    <slot>rectangleSelected(astro::image::ImageRectangle)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>snowgui::coolercontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">coolercontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::filterwheelcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">filterwheelcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::mountcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">mountcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::ccdcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">ccdcontrollerwidget.h</header>
   <container>1</container>
   <slots>
    <signal>imageReceived()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>snowgui::guideportcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">guideportcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::focusercontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">focusercontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::adaptiveopticscontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">adaptiveopticscontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::focusingcontrollerwidget</class>
   <extends>QWidget</extends>
   <header location="global">focusingcontrollerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::FocusingProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">focusingprogresswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>snowgui::FocusElementStack</class>
   <extends>QStackedWidget</extends>
   <header location="global">focuselementstack.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>imageReceived()</slot>
  <slot>rectangleSelected(astro::image::ImageRectangle)</slot>
 </slots>
</ui>
